### PR TITLE
CZML extractor: timezone issue #817

### LINKED ISSUE #817: CZML extractor: timezone issues (clock.Interval and currentTime not tz-aware)
🐞 **Problem**

The ISO8601 timestamps in the `clock.interval` and `currentTime` fields by the czml extractor are missing the 'Z' character indicating UTC. As a result, when the users locale is UTC+1 the CZMLWidget will start one hour too early (so that the satellites will only be visible after one hour passed in the simulation.

[Example notebook](https://gist.github.com/kerel-fs/6cdcfe33b933329f807382b0c5bbf1a9) [Binder](https://mybinder.org/v2/gist/kerel-fs/6cdcfe33b933329f807382b0c5bbf1a9/master)

🖥 **Envorionment**
* `pip freeze`
  - `poliastro==0.13.post0.dev0`

💡 **Possible solutions**

- option A: Extend `czml3.types.TimeInterval` to have also a `_value` like `IntervalValue` has. Then use `czml3.types.TimeInterval` instead of `czml3.types.IntervalValue` in [extract_czml.py#L174-L181](https://github.com/Juanlu001/poliastro/blob/1ddf2d5a8082b7e7d191b0c3ae8de2839b3f51a8/src/poliastro/czml/extract_czml.py#L174-L181):
  ```python
  pckt = Preamble(
      name="document_packet",
      clock=IntervalValue(
          start=self.start_epoch.value,
          end=self.end_epoch.value,
          value=Clock(currentTime=self.start_epoch.value, multiplier=60),
      ),
  )
  ```
- option B: Copy datetime formatting code from `czml3.types.TimeInterval` into the relevant section in `extract_czml.py`

A similar issue was fixed in #783.

### PR DESCRIPTION
**Pull request by sundeshgupta**
Fixes #817. 

I have also added a test to check that `clock.interval` and `clock.currentTime` fields by the czml extractor have the 'Z' character indicating UTC.

### CODE REVIEW

**Comment by Sedictious** [comment]
Looks good but I have to test once I get back home!

**Comment by sundeshgupta** [comment]
Sure! No problem :)

**Comment by sundeshgupta** [comment]
I have updated the existing tests. Kindly review this PR.

File: src/poliastro/tests/test_czml.py [hunk 1]
```diff
@@ -70,8 +70,8 @@ def test_czml_add_orbit():
     "version": "1.0",
     "name": "document_packet",
     "clock": {
-        "interval": "2013-03-18T12:00:00.000/2013-03-18T23:59:35.108",
-        "currentTime": "2013-03-18T12:00:00.000",
+        "interval": "2013-03-18T12:00:00Z/2013-03-18T23:59:35Z",
+        "currentTime": "2013-03-18T12:00:00Z",
```
Comment (Sedictious) [inline src/poliastro/tests/test_czml.py]: I think there isn't any reason not to convert the rest of the properties (like ``epoch``) to the ISO8601 format.

File: src/poliastro/tests/test_czml.py [hunk 2]
```diff
@@ -645,3 +645,45 @@ def test_czml_invalid_orbit_epoch_error():
         "ValueError: The orbit's epoch cannot exceed the constructor's ending epoch"
         in excinfo.exconly()
     )
+
+
+@pytest.mark.skipif("czml3" not in sys.modules, reason="requires czml3")
+def test_czml_timezone():
```
Comment (Sedictious) [inline src/poliastro/tests/test_czml.py]: Isn't this test a bit redundant since it's guaranteed that the timezone format will be tested in every other test anyway?

File: src/poliastro/tests/test_czml.py (hunk 1 above)
Comment (sundeshgupta) [reply src/poliastro/tests/test_czml.py]: I'll update the commit to convert the rest of the properties to ISO8601 format.

File: src/poliastro/tests/test_czml.py (hunk 2 above)
Comment (sundeshgupta) [reply src/poliastro/tests/test_czml.py]: Yes. I also think that the test is redundant. Should I delete this test?

**Comment by sundeshgupta** [comment]
I have converted `epoch` to ISO8601 format. Please tell if should delete `test_czml_timezone()`.

File: src/poliastro/tests/test_czml.py (hunk 2 above)
Comment (Sedictious) [reply src/poliastro/tests/test_czml.py]: I think it would be better if you did since it doesn't test for any specific edge-case or other functionality

Comment (sundeshgupta) [reply src/poliastro/tests/test_czml.py]: I have removed the test. Please tell if there is anything else to update.

Comment (Sedictious) [reply src/poliastro/tests/test_czml.py]: Looks good! Could you perhaps squash and rebase?

Comment (sundeshgupta) [reply src/poliastro/tests/test_czml.py]: I have tried to squash the commits and rebase to master. Please check if I did it correctly.

**Comment by Juanlu001** [comment]
Thanks a lot @sedictious for reviewing this, and @sundeshgupta for the pull request! Merging :muscle:
